Formula/gitlab-time-tracker.rb

```diff
@@ -0,0 +1,19 @@
+require "language/node"
+
+class GitlabTimeTracker < Formula
+  desc "Command-line interface for GitLab's time tracking feature"
+  homepage "https://github.com/kriskbx/gitlab-time-tracker"
+  url "https://registry.npmjs.org/gitlab-time-tracker/-/gitlab-time-tracker-1.7.27.tgz"
+  sha256 "a29b7be0069201688dab67280fa681a1698db80f1f2abab18af962bedf960871"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"gtt", "--version"
```

Comment:
Because of the way homebrew installs npm packages with all the symlinks, it seems the npm package, and more specifically the node_module used to define the subcommands (https://www.npmjs.com/package/commander), the folder where the files are, isn't correctly found inside commander.

With this Formula:
```
$ gtt config
baseDir:/usr/local/Cellar/gitlab-time-tracker/1.7.27/bin
gtt-config(1) does not exist, try --help
```

With npm install -g:
```
$ gtt config
baseDir:/usr/local/lib/node_modules/gitlab-time-tracker/src
```

For the config subcommand, it needs to find the gtt-config.js file in the src folder.

I have no clue if or how I could fix this. I'll open an issue in the commander package, because my feeling is, it should be fixed in that package.